.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+    "root": true,
+    "extends": "airbnb-base",
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "rules": {
+      "one-var": 0,
+      "one-var-declaration-per-line": 0,
+      "new-cap": 0,
+      "consistent-return": 0,
+      "no-param-reassign": 0,
+      "comma-dangle": 0,
+      "curly": ["error", "multi-line"],
+      "import/no-unresolved": [2, { "commonjs": true }],
+      "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
+      "valid-jsdoc": ["error", {
+        "requireReturn": true,
+        "requireReturnType": true,
+        "requireParamDescription": false,
+        "requireReturnDescription": true
+      }],
+      "require-jsdoc": ["error", {
+          "require": {
+              "FunctionDeclaration": true,
+              "MethodDefinition": true,
+              "ClassDeclaration": true
+          }
+      }]
+    }
+}
+
```

Comment:
Why am I seeing this file again in this PR? This file is already on `develop` isn't it? 🤔

Reply:
i'm not sure i worked directly on this file, maybe it was when i rebased that it was altered.